Doppler.HtmlEditorApi/Infrastructure/Repository.cs

```diff
@@ -24,8 +24,8 @@ public async Task<ContentModel> GetCampaignModel(string accountName, int campaig
 JOIN [User] u ON u.IdUser = ca.IdUser
 WHERE co.IdCampaign = @campaignId  AND u.Email = @accountName AND co.EditorType = 5";
                 var result = await connection.QueryFirstOrDefaultAsync<ContentRow>(databaseQuery, new { campaignId, accountName });
-                var res = JsonSerializer.Deserialize<ContentModel>(result.Meta);
-                return res == null ? null : res;
+                var res = result == null ? null : JsonSerializer.Deserialize<ContentModel>(result.Meta);
+                return res;
```

Comment:
Si colocamos esto en el mismo retorno asi 
```suggestion
                return result == null ? null : JsonSerializer.Deserialize<ContentModel>(result.Meta);
```